### ISSUE TITLE
Steam login process extremely slow when running under Sandboxie

### ISSUE THREAD

**Issue by salehSKL** [sandboxie-plus/Sandboxie#4161]
### Describe what you noticed and did

When I run Steam inside Sandboxie and attempt to log in, the process takes an unusually long time, around 10 to 15 minutes. The login screen remains stuck on "Logging in..." for this duration before finally completing the login.

### How often did you encounter it so far?

every time I attempt to log in to Steam while running it through Sandboxie.

### Expected behavior

Steam should log in within a reasonable amount of time, similar to how it performs outside of Sandboxie, where the login process is almost instantaneous.

![Screenshot (411)](https://github.com/user-attachments/assets/d2008ef6-57ff-41cd-8c4b-99cc3950c3dd)

### Affected program

Steam 

### Download link

https://store.steampowered.com/about/

### Where is the program located?

The program is installed only outside the sandbox.

### Did the program or any related process close unexpectedly?

No, not at all.

### What version of Sandboxie are you running now?

Sandboxie-PIus v1.14.6

### Is it a new installation of Sandboxie?

I just updated Sandboxie from a previous version (I don't remember which one).

### In which sandbox type you have this problem?

In a standard isolation sandbox (yellow sandbox icon).

### Can you reproduce this problem on a new empty sandbox?

I can confirm it also on a new empty sandbox.

### What is your Windows edition and version?

Windows 10 Pro 22H2 64-bit

### In which Windows account you have this problem?

A local account (Standard user)., A local account (Administrator).

### Please mention any installed security software

Avast 

### Sandboxie.ini configuration

```ini
[GlobalSettings]
Template=7zipShellEx
Template=AdobeLicensing
Template=Avast_Antivirus
Template=BitDefenderInternetSecurity
Template=Edge_Fix
Template=OfficeClickToRun
Template=OfficeLicensing
Template=RTSS
Template=WindowsLive
Template=WindowsRasMan

[UserSettings_04CE0136]
SbieCtrl_AutoStartAgent=SandMan.exe -autorun
SbieCtrl_EnableAutoStart=y
BoxGrouping=:DefaultBox

[DefaultBox]
Enabled=y
BlockNetworkFiles=y
RecoverFolder=%{374DE290-123F-4565-9164-39C4925E467B}%
RecoverFolder=%Personal%
RecoverFolder=%Desktop%
BorderColor=#00FFFF,ttl
Template=OpenBluetooth
Template=SkipHook
Template=FileCopy
Template=qWave
Template=BlockPorts
Template=LingerPrograms
Template=AutoRecoverIgnore
ConfigLevel=10
OpenFilePath=C:\Program Files (x86)\Steam
OpenFilePath=C:\Program Files (x86)\Steam\steam.exe
OpenFilePath=C:\
```

**Comment by akurpukov** [comment]
I have issue with Steam too. Actually with Valve Anti Cheat. I can play no more than 10 minutes, after this I get disconnected with error message "Client timed out." Relaunching game won't help, same error message instantly when try connect to server. Only relaunching Steam helps, but again no more than 10 minutes. BUT, I can play as long as I want if I'm only one player on the server. After disconnect and opening game console, there some text releated to VAC error code 5. There no info on Steam web-sites releated to this error code 5 (five).

I have Drop Admin Rights and make apps think they have admin rights enabled. Game is Left 4 Dead 2, but I sure in other Valve's games will be same problem.

It is possible to play VAC-enabled online games in sandbox? I read many other issues releated to Steam but they all don't solve problem. Issue releated to VAC or Steam Client Service, it can't perform some actions/checks inside sandbox.

**Comment by offhub** [comment]
> It is possible to play VAC-enabled online games in sandbox?

No, it is not possible to play VAC-enabled online games in a sandbox. Even if you could manage it, there’s no guarantee that your account wouldn’t get banned.

**Comment by Relax0535** [comment]
I'm having this issue as well, wondering if anyone has found a fix in the past 2 years since this issue was created? It happens for me in both Win11 24H2 and a Win10 21H2 VM. For me it's around a 10x increase in time for steam to launch and go through the whole login process, taking ~20s instead of ~2s. Then even after steam is launched everything is very slow, for example clicking the button to install a game can take 5-10s to get the popup to confirm and select where to install to, then even downloading itself is extremely slow going down to below 1MB/s speeds compared to 80MB/s which I get unsandboxed. It happens in every type of box, from application compartment to security hardened. I've tried using trace logging but I don't see anything that's failing getting spammed and I don't know what else to look for.

Any help with this would be appreciated!

**Comment by Relax0535** [comment]
I've found the fix for this, you need to add "StartService=Steam Client Service" to your sandbox ini config. This makes it so the steam service will run in the sandbox, I don't know why steam is slowed down so much without it, but it is how it is.
